pipeline-trigger/.dockerignore

```diff
@@ -0,0 +1,92 @@
+# Created by https://www.gitignore.io/api/node
```

Comment:
Why is the `.dockerignore` file necessary?
In the `Dockerfile`  you only copy the source and config files you need.

Reply:
@andreas-bauer This is absolutely necessary for better docker build performance. Without a `.dockerignore` the **whole** directory is sent to the Docker daemon regardless of the `COPY` commands in the `Dockerfile`. Therefore it is good practice to exclude for example the `node_modules` or `dist` folder from the docker build context (see https://docs.docker.com/engine/reference/builder/#usage for more details).